helm/publisher/templates/web/service.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Release.Name }}
+  labels:
+    app: {{ .Release.Name }}
+{{ if .Values.service.annotations}}
+  annotations:
+    {{- range $key, $value := .Values.service.annotations }}
+    {{ $key }}: {{ $value }}
+    {{- end }}
+{{- end }}
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+    - port: {{ .Values.service.port }}
+      targetPort: {{ .Values.nginxPort }}
+      protocol: TCP
+      name: https
```

Comment:
HTTPS will be terminated at the load balancer, with nginx exposing plain HTTP, so I'd change this to `http` to match.

Reply:
ok, I will modify that, I guess it is the same in other services merged.